extensions/guacamole-auth-totp/src/main/java/org/apache/guacamole/auth/totp/user/TOTPUser.java

```diff
@@ -82,13 +106,15 @@ public User getUndecorated() {
     public void setAttributes(Map<String, String> attributes) {
 
         // Create independent, mutable copy of attributes
-        attributes = new HashMap<String, String>(attributes);
-
-        // Do not expose any TOTP-related attributes outside this extension
-        attributes.remove(TOTP_KEY_SECRET_ATTRIBUTE_NAME);
```

Comment:
This will still need to be removed in the case that we aren't setting `TOTP_KEY_SECRET_ATTRIBUTE_NAME` to anything, or it will be possible for users to manually set the value of their TOTP key.

Reply:
That's what I do below this point, right?  It's just that, instead of using the `remove()` method and sending the value nowhere, I'm assigning it to a `String` for the purpose of checking it.  But it never gets added back or left on.  Correct?  Or am I missing something?

Reply:
Oh, I'm missing it...I see it, fixing it.